_analyzers/supported-analyzers/standard.md

```diff
@@ -26,41 +29,41 @@ PUT /my_standard_index
     "properties": {
       "my_field": {
         "type": "text",
-        "analyzer": "standard"  
+        "analyzer": "standard"
       }
     }
   }
 }
 ```
 {% include copy-curl.html %}
 
-## Parameters
+---
 
-You can configure a `standard` analyzer with the following parameters.
+## Parameters
 
-Parameter | Required/Optional | Data type | Description
-:--- | :--- | :--- | :--- 
-`max_token_length` | Optional | Integer | Sets the maximum length of the produced token. If this length is exceeded, the token is split into multiple tokens at the length configured in `max_token_length`. Default is `255`.
-`stopwords` | Optional | String or list of strings | A string specifying a predefined list of stopwords (such as `_english_`) or an array specifying a custom list of stopwords. Default is `_none_`.
-`stopwords_path` | Optional | String | The path (absolute or relative to the config directory) to the file containing a list of stop words.
+The `standard` analyzer supports the following parameters:
 
+| Parameter | Type | Default | Description |
```

Comment:
Most of the documentation pages use `Data type` instead of `Type`. Eg: https://github.com/opensearch-project/documentation-website/pull/9479/files

Let's stick to a single nomenclature across documentations - either `type` or `data type` - either is fine, as long as it is consistent.

Reply:
@sandeshkr419 thats updated now across the repo, All "Data Type" changed to "Data type"